src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$BodyFont: arial, helvetica, sans-serif;
+$FontSize: 30px;
+$DaySize: 100px;
+$DayGap: 1px;
+$TotalSize: (7 * $DaySize);
+$TotalGap: 7 * $DayGap;
```

Comment:
It's better to name all variables with small letters at the start. It's a common rule for `camelCase`.